docs/metadata.md

```diff
@@ -0,0 +1,72 @@
+# Adding metadata
+
+## Adding binaries
+
+Every part may build a binary that may need to be exposed in the system, with
+that in mind, the way to expose it would be to use the **binaries** keyword
+and declare each binary and its functionality. So, if there were two binaries,
+one called *app1* and another called *app2*, the extract would look like
+
+```yaml
+binaries:
```

Comment:
In master this stuff has moved to 2.0 syntax-- no `binaries` or `services` anymore, only apps.